Add various extra resources (#29)
osucord/resources: content.md

### Replay and Score Analysis
- [Danser-Go](https://github.com/Wieku/danser-go) - Render your replays locally
- [o!rdr](https://ordr.issou.best/) - Render your replays online (powered by Danser-Go!)
- [Rewind](https://github.com/abstrakt8/rewind) - Analyze replays in meticulous detail
- [osu! Miss Analyzer](https://github.com/ThereGoesMySanity/osuMissAnalyzer) - Given a replay, displays where the misses happened (also available as a Discord bot)

> - [osu-aim-analyzer](https://github.com/rgbeing/osu-aim-analyzer) - An alternative replay analysis tool to help analyze aim mistakes

Reviewer: ```suggestion
- [osu-aim-analyzer](https://github.com/rgbeing/osu-aim-analyzer) - A replay analysis tool to help analyze aim mistakes
```

- [osuck](https://skins.osuck.net/) - Website to get skins for all game modes
- [Skinship's skin compendium](https://compendium.skinship.xyz/) - Easy to search archive of all osu! skins posted in the Completed Skins forum
- [osuskins](https://osuskins.net/) - Compendium of osu! skins, especially from top players

> - [OsuSkinMixer](https://github.com/rednir/OsuSkinMixer) - A tool to mix various elements of skins within seconds

Reviewer: ```suggestion
- [OsuSkinMixer](https://github.com/rednir/OsuSkinMixer) - A tool to easily mix various elements of skins
```

- [osu! Web Beatmap Viewer](https://preview.tryz.id.vn/) - View a beatmap in an editor-like interface in the browser
- [Agka's SV Tools](https://zardoru.github.io/sv-tools/) - Tools to help create and edit Scroll Velocities for the osu!mania mode
- [ArrowVortex](https://arrowvortex.ddrnl.com/) - Alternative map editor for various VSRGs
---
### Browser Extensions
- [osuplus](https://osu.ppy.sh/community/forums/topics/408541?n=1) - Extension that adds extra features to the osu! website
- [score inspector](https://github.com/darkchii/score-inspector-extension) - A userscript that adds more stats to user profiles on the official osu! website (Extension compatible with the [osu! scores inspector](https://score.kirino.sh/))
- [osu! preview](https://github.com/JerryZhu99/osu-preview) - Preview a beatmap without downloading it
- [osu! subdivide nations](https://github.com/Cavitedev/osu-subdivide-nations) - Display the regional flags and names of players on the official osu! website as well as other unofficial sites (Extension compatible with [osu! World](https://osuworld.octo.moe/))
- [ezpp!](https://github.com/oamaok/ezpp) - Calculate how much PP a map is worth for a particular score

> - [osu-color-changer](https://userstyles.world/style/1767/osu-color-changer) - various colour and stylistic changes for the osu! website, featuring many customization options

Reviewer: ```suggestion
- [osu-color-changer](https://userstyles.world/style/1767/osu-color-changer) - A userstyle allowing various colour and stylistic changes for the osu! website, featuring many customization options
```
